framework/src/main/java/org/tron/core/services/event/HistoryEventService.java

```diff
@@ -54,7 +54,10 @@ private void syncEvent() {
     try {
       long tmp = instance.getStartSyncBlockNum();
       long endNum = manager.getDynamicPropertiesStore().getLatestSolidifiedBlockNum();
-      while (tmp <= endNum) {
+      while (tmp < endNum) {
```

Comment:
if tmp == endNum, then the while loop exists, and the startNum will be set to tmp - 1. For example, when tmp is 108, endNum is 109, and the while loop will be executed, then when tmp is 109 and endNum is 109, startNum is 109 -1 = 108, for the block 109, it will not be run the codes "realtimeEventService.flush(blockEvent, false); solidEventService.flush(blockEvent);" will not be executed, is that any problem?